src/api/liveSpecs.ts

```diff
@@ -0,0 +1,69 @@
+import { PostgrestResponse } from '@supabase/postgrest-js';
+import pLimit from 'p-limit';
+import {
+    handleFailure,
+    handleSuccess,
+    supabaseClient,
+    TABLES,
+} from 'services/supabase';
+import { Entity } from 'types';
+
+export interface LiveSpecsExtQuery_ByCatalogName {
+    catalog_name: string;
+    spec_type: string;
+    spec: any;
+}
+
+export const getLiveSpecsByCatalogName = async (
+    catalogName: string,
+    specType: Entity
+) => {
+    const data = await supabaseClient
+        .from(TABLES.LIVE_SPECS_EXT)
```

Comment:
`src/api/liveSpecsExt.ts` was introduced while this PR was under development. The queries specified in this file can be performed on both `live_specs` and `live_specs_ext`. If memory serves, you would prefer for _tables_ to be queried whenever possible, correct? If so, I can swap the data source for the query. If not, I can move these queries into `src/api/liveSpecsExt.ts`.

Reply:
I'd stick with the queries you have - we can decide if we wanna move to the tables later.